### PR TITLE
Flood Multisenson PAT02-A/B/C

### PR DESCRIPTION
**Pull request by schmidmuc**
Took the information from the datasheet of the PAT02 of Philio

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/database/philio/PAT02 [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+			<Maximum>100</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Basic Set OFF Level</Label>
+			<Help lang="en"><![CDATA[Setting the BASIC command value.<br/>
+			    When the flood trigger off (0x00), send the BASIC CC to the group 2]]></Help>
+		</Parameter>
+		<Parameter>
+			<Index>2</Index>
+			<Type>byte</Type>
+			<Default>0xFF</Default>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/philio/PAT02]: I think this needs to be decimal

File: bundles/binding/org.openhab.binding.zwave/database/philio/PAT02 [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+			<Help lang="en"><![CDATA[The interval time for auto reporting the flood state.<br/>
+				<ul><li>0 = Turn off auto report temperature.</li>
+				<li>1-127 = Number of ticks. The default value is 12. The tick time can be set by configuration No. 20.</li></ul>]]></Help>
+		</Parameter>
+		<Parameter>
+			<Index>20</Index>
+			<Type>byte</Type>
+			<Default>30</Default>
+			<Minimum>0</Minimum>
+			<Maximum>254</Maximum>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/philio/PAT02]: Shouldn't this be 127 given the information in the help?

File: bundles/binding/org.openhab.binding.zwave/database/philio/PAT02 (hunk 1 above)
Comment (schmidmuc) [inline bundles/binding/org.openhab.binding.zwave/database/philio/PAT02]: You're right

File: bundles/binding/org.openhab.binding.zwave/database/philio/PAT02 (hunk 2 above)
Comment (schmidmuc) [inline bundles/binding/org.openhab.binding.zwave/database/philio/PAT02]: It has to be 254, as the validity range is 0 to 0xFF

**Comment by watou** [comment]
@cdjackson is the file name "PAT02" correct?

**Comment by cdjackson** [comment]
Well spotted - it should be .xml!  @schmidmuc can you correct please and produce another PR.

**Comment by schmidmuc** [comment]
The type of the sensor is "PAT02", so I'm not sure where there is the failure.

**Comment by watou** [comment]
@schmidmuc could you make a new PR that renames the file PAT02 to pat02.xml, and any references to it are also changed to the new name? You will notice the other filenames in the philio directory.  Also, is this file referenced from products.xml?

**Comment by schmidmuc** [comment]
Thanks for explenation... Done.

**Pull request merged**